src/abstractdataframe/reshape.jl

```diff
@@ -212,7 +212,7 @@ function unstack(df::AbstractDataFrame, colkey::Int, value::Int)
         j = Int(keycol.refs[k])
         i = rowkey[k]
         if i > 0 && j > 0
-            if nowarning && !isna(df2[j][i])
+            if nowarning && !isna.(df2[j][i])
```

Comment:
`df2[j][i]` should be a scalar, right? Otherwise unary `!` would throw a depwarn on 0.6 (not to mention it wouldn't work in this `Bool`-only context).